python-threatexchange/threatexchange/exchanges/impl/ncmec_api.py

```diff
@@ -303,12 +310,25 @@ def log(event: str) -> None:
                         # Our entry estimatation (based on the cursor parameters)
                         # occasionally seem to over-estimate
                         log(f"est {entry.estimated_entries_in_range} entries")
-                elif i % 100 == 0:
+
+                updates.extend(entry.updates)
+
+                if i % 100 == 0:
```

Comment:
blocking: by change this from `elif` to `if`, I think it will now print the large update warning every update, which is incorrect, no?

Reply:
it would print for the 0th, which we would not want. I updated this to be (i + 1) % 100 == 0, so it's every 100th iteration

we need to extend `updates` every time and regardless of `i` so this was cleaner than other things I thought of, but please suggest alternatives